### ISSUE TITLE
0.7.6: pytest warnings

### ISSUE THREAD

**Issue by kloczek** [edgewall/genshi#62]
```console
+ /usr/bin/pytest -ra
=========================================================================== test session starts ============================================================================
platform linux -- Python 3.8.12, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6
plugins: cases-3.6.8, mock-3.7.0
collected 57 items

genshi/filters/tests/test_html.py .........................................................                                                                          [100%]

============================================================================= warnings summary =============================================================================
genshi/filters/html.py:357
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6/genshi/filters/html.py:357: DeprecationWarning: invalid escape sequence \s
    u'[Uu][Rr\u0280][Ll\u029F]\s*\(([^)]+)').finditer

genshi/filters/i18n.py:1119
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6/genshi/filters/i18n.py:1119: DeprecationWarning: invalid escape sequence \[
    data = data.replace('[', '\[').replace(']', '\]')

genshi/filters/i18n.py:1119
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6/genshi/filters/i18n.py:1119: DeprecationWarning: invalid escape sequence \]
    data = data.replace('[', '\[').replace(']', '\]')

genshi/filters/i18n.py:1174
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6/genshi/filters/i18n.py:1174: DeprecationWarning: invalid escape sequence \[
    part.replace('\[', '[').replace('\]', ']'),

genshi/filters/i18n.py:1174
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6/genshi/filters/i18n.py:1174: DeprecationWarning: invalid escape sequence \]
    part.replace('\[', '[').replace('\]', ']'),

genshi/template/interpolation.py:33
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6/genshi/template/interpolation.py:33: DeprecationWarning: Flags not at the start of the expression '[uU]?[rR]?("""|\\\'\\\'\\' (truncated)
    token_re = re.compile('%s|%s(?s)' % (

genshi/path.py:663
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.6/genshi/path.py:663: DeprecationWarning: invalid escape sequence \d
    _tokenize = re.compile('("[^"]*")|(\'[^\']*\')|((?:\d+)?\.\d+)|(%s)|([^%s\s]+)|\s+' % (

-- Docs: https://docs.pytest.org/en/stable/warnings.html
====================================================================== 57 passed, 7 warnings in 0.27s ======================================================================
```

**Comment by kloczek** [comment]
Just tested build of 0.7.7 and still I see one warning
```console
============================================================================= warnings summary =============================================================================
genshi/filters/html.py:357
  /home/tkloczko/rpmbuild/BUILD/genshi-0.7.7/genshi/filters/html.py:357: DeprecationWarning: invalid escape sequence \(
    u'[Uu][Rr\u0280][Ll\u029F]%s*\(([^)]+)' % (r'\s')).finditer

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
====================================================================== 57 passed, 1 warning in 0.27s =======================================================================
```
Thank you 🙂

**Comment by hodgestar** [comment]
Oh dear, how did that one sneak through?

**Comment by kloczek** [comment]
I'm only messanger .. 🙂

**Comment by hodgestar** [comment]
Fixed in #76.
